bootcamp-api.API/Program.cs

```diff
@@ -15,7 +13,7 @@
     options.AddDefaultPolicy(
         policy =>
         {
-            policy.WithOrigins("http://localhost:4200");
+            policy.WithOrigins("*");
```

Comment:
This _could_ be set to your live site URL. Typically, this is a configuration that is set based on environment. * is fine for our dev but normally this is explicitly set.